src/server/metrics/image.metrics.ts

```diff
@@ -168,12 +187,25 @@ export const imageMetrics = createMetricProcessor({
           FROM "ImageReaction" ir
           GROUP BY ir."imageId"
         ) r ON q.id = r.id
+        LEFT JOIN (
+          SELECT
+            ici."imageId" AS id,
+            COUNT(*) AS collected_count,
+            SUM(IIF(img."createdAt" >= (NOW() - interval '365 days'), 1, 0)) AS year_collected_count,
+            SUM(IIF(img."createdAt" >= (NOW() - interval '30 days'), 1, 0)) AS month_collected_count,
+            SUM(IIF(img."createdAt" >= (NOW() - interval '7 days'), 1, 0)) AS week_collected_count,
+            SUM(IIF(img."createdAt" >= (NOW() - interval '1 days'), 1, 0)) AS day_collected_count
```

Comment:
These "createdAt"s should be from the collection item instead of the image since we're tracking when the collecting happened, not when the image was created. The same goes for all other instances...